code/datums/gamemodes/campaign/faction_stats.dm

```diff
@@ -474,7 +493,7 @@ GLOBAL_LIST_INIT(campaign_mission_pool, list(
 			if(current_mode.current_mission?.mission_state != MISSION_STATE_FINISHED)
 				to_chat(user, "<span class='warning'>Current mission still ongoing!")
 				return
-			if(!(stats_flags & MISSION_SELECTION_ALLOWED))
+			if(!(stats_flags & CAMPAIGN_TEAM_MISSION_SELECT_ALLOWED))
 				to_chat(user, "<span class='warning'>The opposing side has the initiative, win a mission to regain it.")
```

Comment:
LUMIIIIIIIIIIIIIIII

Reply:
gonna do it every day until you like it.